github/repos_codeowners.go

```diff
@@ -0,0 +1,41 @@
+package github
```

Comment:
Please add the standard copyright header with the year 2022 here before the `package` line with a blank line separating the two.

Reply:
Done! Thanks, forgot about that.